seodongheee2/services/userServices.js

```diff
@@ -9,13 +13,15 @@ const signUp = async (name, email, profileImage, password) => {
     err.statusCode = 409;
     throw err;
   }
+
+  const saltRounds = 12;
+  const hashedPassword = await makeHash(password, saltRounds);
```

Comment:
굳이 makeHash 함수를 만들지 않고 아래처럼 바로 사용할 수 있습니다.

```suggestion
  const hashedPassword = await bcrypt.hash(password, saltRounds);
```